library/src/main/java/ru/noties/markwon/spans/SpannableTheme.java

```diff
@@ -173,6 +173,14 @@ private static int resolve(Context context, @AttrRes int attr) {
     // by default, text color with `HEADING_DEF_BREAK_COLOR_ALPHA` applied alpha
     protected final int headingBreakColor;
 
+    // by default, whatever typeface is set on the TextView
+    protected final Typeface headingTypeface;
+
+    // by default, we use standard multipliers from the HTML spec (see HEADING_SIZES for values).
+    // this library supports 6 heading sizes, so make sure the array you pass here has 6 elements.
+    // note also that these values are in PIXELS, so you'll need to convert from SP first.
+    protected final float[] headingTextSizes;
```

Comment:
I think we should follow the relative sizes pattern (like `HEADING_SIZES` does), so we provide relative values instead of absolute

Reply:
Yeah, I think this comes from a difference of approach: I'm building something where the markdown is provided by my code on the server, so I know exactly what it's going to be, and I know exactly what font size I need it to be.  So it seems a little backwards to specify the sizes as multipliers.  But I can totally understand the standpoint of the "I'm rendering arbitrary user-provided markdown" and multipliers probably make more sense in that case.  

Regardless, changed.